README.md

```diff
@@ -19,8 +19,8 @@ Dependency Analytics only accesses your manifest files to analyze your applicati
 vulnerability report.
 
 **IMPORTANT:**
-<br >Currently, Dependency Analytics only supports projects that use Maven (`mvn`), Node (`npm`), Golang (`go mod`) and
-Python (`pip`) ecosystems, and base images in `Dockerfile`.
+<br >Currently, Dependency Analytics only supports projects that use Maven (`mvn`), Node (`npm`), Golang (`go mod`) ,
```

Comment:
Towards the end of the line, it looks like there is a space between the last parenthesis bracket and the comma, which there shouldn't be.  The comma goes directly after, like so:

`... Golang (go mod),`